cli/packages/api/api.go

```diff
@@ -161,6 +180,22 @@ func CallGetAllWorkSpacesUserBelongsTo(httpClient *resty.Client) (GetWorkSpacesR
 		return GetWorkSpacesResponse{}, fmt.Errorf("CallGetAllWorkSpacesUserBelongsTo: Unsuccessful response:  [response=%v]", response)
 	}
 
+	// Call the organization API
+	orgResponse, err := CallGetAllOrganizations(httpClient)
+	if err != nil {
+		return GetWorkSpacesResponse{}, err
+	}
+
+	// Update organization names in workspacesResponse
```

Comment:
While this works, I think a new backend API or existing one with org name attached will be better. Have you thought about this?

Reply:
We can consider a new api. I didn't wanted to change existing api as it may be a breaking change